rdmo/accounts/tests/test_views.py

```diff
@@ -332,3 +332,38 @@ def test_signup_next(db, client):
 
     assert response.status_code == 302
     assert response.url == '/about/'
+
+
+def test_token_get_for_user(db, client):
```

Comment:
maybe I should add the test `pytest.parametrize` setting boolean toggle here as well for `ACCOUNT_ALLOW_USER_TOKEN`, like in https://github.com/rdmorganiser/rdmo/blob/c4946bb72ac0826b415daf817bce781931938298/rdmo/accounts/tests/test_views.py

Reply:
Yes, I think that would be better, since we run the tests always with the kind-of-generic settings in `testing/config/settings/base.py`.

Reply:
Otherwise the setting needs to be enabled there.

Reply:
Which you already did :see_no_evil: